### DISCUSSION TITLE
Change date and time on Master node Security Onion

### DISCUSSION THREAD

**Discussion by admarnelson** [Security-Onion-Solutions/securityonion#6821] [Unsupported Versions] [answered]
Hi
I want set real time on Master node security onion. I tried with some commands but no success:

# sudo timedatectl  --- working well showing my current time
# sudo timedatectl  --- working command but answer is FAILED AUTOMATIC TIME SYNCHRONIZATION IS ENABLE

example real current time is 8:34:00 PM
on Sec Onion Master node is 7:34:00 PM

what to do ? help please

**Comment by abesinger** [comment]
I suspect that what you actually want to do is change the timezone for your system, not change the clock time. But to directly answer your question, the reason you are getting that message is that your system has NTP (network time protocol) enabled.  `timedatectl` will not let you change the time when the time is being synchronized from the network, which is enabled by default on SecurityOnion (and on many Linux distros).  If you really want to set the time manually you have to disable NTP.

`timedatectl status` will show a number of things, including the system timezone and status of NTP.  Take a look at the timezone in the output of that first, and see if it's what you want, and then take it from there.  `timedatectl set-timezone` will change the timezone for you.

**Comment by TOoSmOotH** [comment]
It is set to UTC on purpose. As a security practitioner you should get used to using UTC time. You can change the display time in hunt to your time zone and it will do the math for you but all your logs and systems should be set to UTC.

**Comment by dougburks** [comment, marked as answer]
Please see:
https://docs.securityonion.net/en/2.3/timezones.html
